contrib/devtools/README.md

```diff
@@ -1,38 +1,31 @@
 Contents
 ========
-This directory contains tools for developers working on this repository.
+Tools for developers working on this repository.
 
 github-merge.py
 ===============
-
-A small script to automate merging pull-requests securely and sign them with GPG.
+This script is used to automate the merging of "pull requests" securely, and signing them with GPG.
 
 For example:
 
-  ./github-merge.py 3077
+    ./github-merge.py 3077
 
-(in any git repository) will help you merge pull request #3077 for the
-bitcoin-core/bitcoincore.org repository.
+(in any git repository) will help you merge pull request #3077 for the bitcoin-core/bitcoincore.org repository.
 
 What it does:
-* Fetch master and the pull request.
-* Locally construct a merge commit.
-* Show the diff that merge results in.
-* Ask you to verify the resulting source tree (so you can do a make
-check or whatever).
-* Ask you whether to GPG sign the merge commit.
-* Ask you whether to push the result upstream.
-
-This means that there are no potential race conditions (where a
-pullreq gets updated while you're reviewing it, but before you click
-merge), and when using GPG signatures, that even a compromised github
-couldn't mess with the sources.
+  * fetches master, and the pull request
+  * locally constructs a merge commit
+  * shows the diff that the merge results in
+  * asks to verify the resulting source tree (so you can do a make check, etc.)
```

Comment:
I prefer `asks you` in this and the following two bullets, since it's more explicit.